legacyCore/src/main/res/layout-sw300dp/login_image.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ImageView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This would be better suited as a style of image view

Reply:
made into a style